shared/logging.properties

```diff
@@ -0,0 +1,8 @@
+handlers = com.microsoft.azure.appservice.logging.AsyncFileHandler, java.util.logging.ConsoleHandler
+
+# AzMon logging
+com.microsoft.azure.appservice.logging.AsyncFileHandler.level = FINE
+com.microsoft.azure.appservice.logging.AsyncFileHandler.directory = /var/log/diagnosticLogs
+com.microsoft.azure.appservice.logging.AsyncFileHandler.prefix = application.${workerid}.
```

Comment:
>${workerid} [](start = 77, length = 11)

double check with Alok if this is needed

Reply:
This should be good. @ shrirang - can you paste example file name. The format should be like for example : application.92ca53ad8db4fbb93d4d3b7d8ab54dcf8ffecb2d731f25b0e91ad575d7534c3f.2019-04-15.log

---
In reply to: [275581026](https://github.com/Azure-App-Service/java/pull/22#discussion_r275581026) [](ancestors = 275581026)

Reply:
Example: application.dev.2019-04-16.log where dev is the COMPUTERNAME env var